docs/.vuepress/config.js

```diff
@@ -61,7 +72,7 @@ module.exports = {
         // if your docs are in a specific branch (defaults to 'master'):
         docsBranch: 'source',
         // defaults to false, set to true to enable
-        editLinks: false,
```

Comment:
I set these to false so that they don't appear in the Standard and Implementation categories, as they will be dead links. They are afterwards enabled manually in each of the other files.